react/components/SKUSelector/components/SelectorItem.tsx

```diff
@@ -40,22 +42,30 @@ const SelectorItem: FC<Props> = ({
   imageUrl,
   imageLabel,
   isImpossible,
+  heightImg,
+  widthImg,
 }) => {
   const discount = getDiscount(maxPrice, price)
+
+  const rootClasses = classNames(
```

Comment:
why "root"?

Reply:
Because these are the classes of the root element of the component

Reply:
ok. we generally call them "container". This is just a suggestion, but maybe rename it for consistency?